app/helpers/data_getter.py

```diff
@@ -715,7 +715,13 @@ def get_locations_of_events():
             for event in DataGetter.get_live_and_public_events():
                 if not string_empty(event.location_name) and not string_empty(event.latitude) and not string_empty(
                      event.longitude):
-                    names.append(event.location_name.split(',')[0])
```

Comment:
Instead of a request, try using searchable_location_name. 

Request to the API is gonna be very slow without caching